src/ripple/app/ledger/impl/LedgerConsensus.cpp

```diff
@@ -1522,20 +1522,20 @@ class LedgerConsensusImp
                 << mPeerPositions.size () << " nw:" << neededWeight
                 << " thrV:" << threshVote << " thrC:" << threshConsensus;
 
-            for (auto it = closeTimes.begin ()
-                , end = closeTimes.end (); it != end; ++it)
+            for (auto i = closeTimes.begin ()
+                , end = closeTimes.end (); i != end; ++it)
```

Comment:
Hmmm.  `i` may be the best we can do here, since there's already a `closeTime` in scope.